### DISCUSSION TITLE
columns-{x} does not render

### DISCUSSION THREAD

**Discussion by sam-dugmore** [tailwindlabs/tailwindcss#6719] [Help] [answered]
I have recently upgraded to tailwind v3 and I was trying to use the `columns-{x}` feature. Only to find that the only one that works is `columns-4` and is the only one present in the the compiled CSS. 
![image](https://user-images.githubusercontent.com/86777888/147347028-9c5e8166-b67f-4120-8b64-4aa1d5f41b79.png)
I am running it with Symfony and Webpack. As can be seen in the screenshot, other classes have loaded but not the columns ones. Any help would be apprectiated

**Comment by adamwathan** [comment]
Hey! What does your config look like? Make sure your `content` config is properly matching all of your content files.

**Reply by sam-dugmore** [reply]
![image](https://user-images.githubusercontent.com/86777888/147359405-39fd7043-06f7-46b2-bbbc-db9cd7e22798.png)

This is my `tailwind.config.js` and all of my templates live in the templates folder only

**Comment by sam-dugmore** [comment, marked as answer]
The classes weren't loading due to not running the watch command for changes and are not loading in the front end. This is due to the change from purging unused classes to adding the used ones in the build.
